templates/emails/received_identification_suggestion-body-text.html

```diff
@@ -0,0 +1,3 @@
+Você recebeu uma sugestão no seu pedido de identificação:
```

Comment:
```suggestion
Uma nova identificação foi sugerida em sua planta!
```
Como o Alisson disse acho que fica legal, mas dessa forma também me parece bom.